nest-forms-backend/src/files/files.helper.ts

```diff
@@ -141,9 +141,18 @@ export default class FilesHelper {
     try {
       return await this.prisma.files.create(createData)
     } catch (error) {
+      if (error instanceof Error) {
+        throw this.throwerErrorGuard.InternalServerErrorException(
+          ErrorsEnum.DATABASE_ERROR,
+          'Error while saving file to database.',
+          undefined,
+          error,
+        )
+      }
       throw this.throwerErrorGuard.InternalServerErrorException(
         ErrorsEnum.DATABASE_ERROR,
-        `Error while saving file to database. Error: ${<string>error}`,
+        'Error while saving file to database.',
+        <string>error,
       )
     }
```

Comment:
I really really don't like this. It's one line of code and 16 lines of error throwing – not even error handling, as that we don't distinguish different types of errors and use different response – whatever happens, it will throw 500. I don't know how to do it better, but this doesn't seem right. And it's EVERYWHERE.

Can we perhaps have this if - else inside that function and construct the error there accordingly? Sure, it will double up the code for each of those `throwerErrorGuard` helper functions, but that's much more acceptable there, than on multiple places in the code where logic should be the focus.

@MarekBodingerBA do you have any ideas, how we could approach this feasibly?